examples/dreambooth/train_dreambooth_flax.py

```diff
@@ -437,15 +459,22 @@ def collate_fn(examples):
     elif args.mixed_precision == "bf16":
         weight_dtype = jnp.bfloat16
 
+    if args.pretrained_vae_name_or_path:
+        vae_arg, vae_kwargs = (args.pretrained_vae_name_or_path, {"from_pt": True})
```

Comment:
We should upload the flax weights for the VAE, then we won't need to use the `from_pt` arg. Will upload it soon.

Reply:
cool, let me know when this is done and I'll change